### ISSUE TITLE
Bug: CodingMentor Suggests Modifying Test Suite Instead of Fixing Buggy Code

### ISSUE THREAD

**Issue by hemanialaparthi** [GatorEducator/execexam#33]
Occasionally, when the advice model is executed, the output suggests modifying the test suite to pass the build rather than addressing the buggy code causing the test case errors. This behavior could be due to an issue with the prompts being sent to the API

**Comment by PCain02** [comment]
I suggest we send not only the prompts but also the source code to the coding mentor so it can not only have the tests that fail but also the buggy code.

**Comment by boulais01** [comment]
@PCain02 and @hemanialaparthi, this has gone 2 weeks without a status update; would you mind letting us know if there's any progress here?

**Comment by PCain02** [comment]
Yep here is my draft pull request https://github.com/GatorEducator/execexam/pull/48

**Comment by gkapfham** [comment]
Hi @hemanialaparthi is this issue connected to any other issues?

**Comment by gkapfham** [comment]
Hi @PCain02 is your PR finalized now? I had the impression that it was. Correct?
